index.html

```diff
@@ -88,6 +88,7 @@
         openmct.install(openmct.plugins.ExampleImagery());
         openmct.install(openmct.plugins.Timeline());
         openmct.install(openmct.plugins.UTCTimeSystem());
+        openmct.install(openmct.plugins.NonEditableFolder());
```

Comment:
Default plugins should be installed in MCT.js, the ones installed in index.html will only work for this environment, not for other environment using openmct via node modules

Reply:
Ah! I even had it there too and moved it out. Now I know! Thanks!